entity-types/ext-database/summary_metrics.yml

```diff
@@ -0,0 +1,16 @@
+state:
+  goldenMetric: state
+  title: Database State
+  unit: COUNT
```

Comment:
Is it a count?

Reply:
Ah, thanks, I changed it to stateDesc as a string to see the friendly state name